llm/llama.go

```diff
@@ -538,7 +538,7 @@ func (llm *llama) Predict(ctx context.Context, prevContext []int, prompt string,
 		return fmt.Errorf("failed to marshal data: %v", err)
 	}
 
-	endpoint := fmt.Sprintf("http://127.0.0.1:%d/completion", llm.Port)
+	endpoint := fmt.Sprintf("http://127.0.0.1:%d/predict", llm.Port)
```

Comment:
changed this endpoint in the `ollama-runner` to match our endpoint for simplicity